scripts/s3_bench/s3bench.py

```diff
@@ -72,8 +75,17 @@ def create_log(resp, log_file_prefix, client, samples, size):
     :param size: object size
     :return: Path of the log file
     """
-    if not path_exists(LOG_DIR):
-        make_dirs(LOG_DIR)
+    host = kwargs.get("host", None)
+    user = kwargs.get("user", None)
+    pwd = kwargs.get("pwd", None)
+    remote = True if host else False
+    if remote:
+        ret = make_remote_dirs(LOG_DIR, host, user, pwd)
```

Comment:
what if on remote the space is less and we got more logs?

Reply:
Test will fail with No space left. This is applicable for not just remote, for local run also.
We will have to clear the space then.